src/v0/destinations/ga4/transform.js

```diff
@@ -39,16 +39,35 @@ const {
 } = require("./utils");
 
 /**
- * Returns GA4 client_id
+ * checks for hybrid mode
```

Comment:
can move this to common util - can be used across integrations later on

Reply:
updated